samples/Matty/Matty.Server/Startup.cs

```diff
@@ -0,0 +1,166 @@
+                options.ClaimsIdentity.UserNameClaimType = Claims.Name;
+                options.ClaimsIdentity.UserIdClaimType = Claims.Subject;
+                options.ClaimsIdentity.RoleClaimType = Claims.Role;
+
+                // Note: to require account confirmation before login,
+                // register an email sender service (IEmailSender) and
+                // set options.SignIn.RequireConfirmedAccount to true.
+                //
+                // For more information, visit https://aka.ms/aspaccountconf.
+                options.SignIn.RequireConfirmedAccount = false;
+            });
+
+            // OpenIddict offers native integration with Quartz.NET to perform scheduled tasks
+            // (like pruning orphaned authorizations/tokens from the database) at regular intervals.
+            services.AddQuartz(options =>
+            {
+                options.UseMicrosoftDependencyInjectionJobFactory();
+                options.UseSimpleTypeLoader();
+                options.UseInMemoryStore();
+            });
+
+            // Register the Quartz.NET service and configure it to block shutdown until jobs are complete.
+            services.AddQuartzHostedService(options => options.WaitForJobsToComplete = true);
+
+            services.AddOpenIddict()
+
+                // Register the OpenIddict core components.
+                .AddCore(options =>
+                {
+                    // Configure OpenIddict to use the Entity Framework Core stores and models.
+                    // Note: call ReplaceDefaultEntities() to replace the default OpenIddict entities.
+                    options.UseEntityFrameworkCore()
+                           .UseDbContext<ApplicationDbContext>();
+
+                    // Enable Quartz.NET integration.
+                    options.UseQuartz();
+                })
+
+                // Register the OpenIddict server components.
+                .AddServer(options =>
+                {
+                    // Enable the device, verification, token and userinfo endpoints.
+                    options.SetDeviceEndpointUris("/connect/device")
+                           .SetVerificationEndpointUris("/connect/verify")
+                           .SetTokenEndpointUris("/connect/token")
+                           .SetUserinfoEndpointUris("/connect/userinfo");
+
+                    // Mark the "email", "profile" and "roles" scopes as supported scopes.
+                    options.RegisterScopes(Scopes.Email, Scopes.Profile, Scopes.Roles);
+
+                    // Note: this sample only uses the device code flow but you can enable
+                    // the other flows if you need to support implicit, password, etc.
+                    options.AllowDeviceCodeFlow();
+                    options.AllowRefreshTokenFlow();
+
+                    // Note: All device codes have a default lifetime of 10 minutes.
+                    options.SetDeviceCodeLifetime(TimeSpan.FromMinutes(5));
```

Comment:
I see `options.SetDeviceCodeLifetime(TimeSpan.FromMinutes(5));` is still there. Any particular reason why you believe it's necessary? I likely missed something.

Reply:
There is no reason for it, I just didn't understand your previous comment well